tests/python/opinfo/opinfos.py

```diff
@@ -1090,6 +1090,7 @@ def scatter_wrapper(
         ArgumentType.Symbolic,
         ArgumentType.Constant,
     ),
+    supports_direct_bindings=True,
```

Comment:
nitpick: we don't have a functional scatter at this moment. @naoyam is adding limited support, which might not handle all the scatter cases in opinfo. https://github.com/NVIDIA/Fuser/pull/4742